packages/ra-ui-materialui/src/button/SortButton.tsx

```diff
@@ -114,7 +114,7 @@ const SortButton: FC<{ fields: string[]; label?: string }> = ({
                         data-sort={field}
                         key={field}
                     >
-                        {translate(`resources.products.fields.${field}`)}{' '}
+                        {translate(field)}{' '}
```

Comment:
no, this should be `resources.${resource}.fields.${field}`

Reply:
Done